2.describe-data/1.merge-datasets-gct.ipynb

```diff
@@ -135,7 +135,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
```

Comment:
It actually looks like you are correcting an error in the repo! In other words, the base state is incorrect and you are fixing it. Nice!

Reply:
One thing - please delete the last two cells in this notebook (they are blank if you look at the `.ipynb` file). This is also to improve the base repo state! :)

Reply:
nvm, I was wrong. Please ignore this comment as I am being confusing!